.github/PULL_REQUEST_TEMPLATE/version_bump_template.md

```diff
@@ -0,0 +1,15 @@
+Pre-review checklist:
+- [ ] The version number in `pyproject.toml` has been updated
+- [ ] `__version__` in `src/nuanced/__init__.py` has been updated
+- [ ] `uv.lock` has been updated by running `uv sync`
+- [ ] `CHANGELOG.md` has been updated
+
+Post-merge checklist:
+- [ ] Create GitHub Release
+- [ ] Publish package to PyPI
```

Comment:
I don't know that it's totally necessary but do we want to be explicit and mention both TestPyPI and PyPI here?

Reply:
Good call! Better to err on the side of being explicit, I think